docs/pages/reference/account-kit/wallet-client/functions/listAccounts.mdx

```diff
@@ -0,0 +1,74 @@
+---
+# This file is autogenerated
+title: listAccounts
+description: Overview of the listAccounts method
+slug: wallets/reference/account-kit/wallet-client/functions/listAccounts
+---
+
+Lists all smart accounts for a given signer using the wallet API client.
+
+## Import
+
+```ts
+import { listAccounts } from "@account-kit/wallet-client";
+```
+
+## Usage
+
+```ts
+// Fetch the first page of accounts
+const firstPage = await client.listAccounts({
+  signerAddress: "0x123...",
+  limit: 10,
+});
+
+// If an 'after' cursor exists, use it to fetch the next page
+const nextPage = await client.listAccounts({
+  signerAddress: "0x123...",
+  limit: 10,
+  after: firstPage.meta.after,
+});
+```
+
+## Parameters
+
+### client
```

Comment:
do you think it's confusing that the examples show using the actions on the client, but the descriptions show the params that are curried in (client, signer, etc)?

Reply:
I think this is a limitation of the current docsgen - i.e. there are other curried things we currently display: 
- https://www.alchemy.com/docs/wallets/reference/account-kit/infra/functions/alchemyActions
- https://www.alchemy.com/docs/wallets/reference/account-kit/smart-contracts/functions/installPlugin

I would ideally like to restructure our generated docs away from individual functions and more towards interfaces, but that's a bigger undertaking than what's happening in this PR.